### PR TITLE
19885-incorrect-amalgamation-date-format

### PR DESCRIPTION
**Pull request by jamespaologarcia**
*Issue #:* /bcgov/entity#19885

Changed the format of the date in the certificate of amalgamation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/report-templates/certificateOfAmalgamation.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
                     <div class="cert-separator"></div>
                     <div class="cert-issued-by">
                         <div>Issued under my hand at Victoria, British Columbia</div>
-                        <div>on {{ report_date_time }}</div>
+                        <div>on {{ report_date }}</div>
```
Comment (severinbeauvais) [inline legal-api/report-templates/certificateOfAmalgamation.html:42]: This is not the same as [Certificate of Incorporation](https://github.com/bcgov/lear/blob/47dc70f37c6e44f11b86ba6a733c60da83fec5ee/legal-api/report-templates/certificateOfIncorporation.html#L36), but then several variables in here are different so maybe this is OK. I don't know.

Comment (severinbeauvais) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: James, can you please test this on a past amalgamation and verify that this date is the same as the effective date (not today's date)? Please post results here. Thanks.

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: `report_date` will print current date. thats not what Certificate of IA does. replace `report_date` with `business.formatted_founding_date`

Comment (jamespaologarcia) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: Confirmed with Yui. I.e. the date of amalgamation is Feb 16, and the certificate is generated today, the issuance date should be Feb 21 on a date(not datetime) format.

**Review by vysakh-menon-aot — APPROVED** [review]

File: legal-api/report-templates/certificateOfAmalgamation.html (hunk 1 above)
Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: Certificate of Incorporation has business.formatted_founding_date

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: if its wrong then we need to revisit other certificates as well

Comment (severinbeauvais) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: > Certificate of Incorporation has business.formatted_founding_date

Why is that different from what Yui said?

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: will check with Yui

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: Yui confirmed we need to follow IA. please update as i mentioned above

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: certificates which shows `report_date` might be a bug. Yui will follow up with Naveen on that

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:42]: yes. just follow Certificate of IA

File: legal-api/report-templates/certificateOfAmalgamation.html [hunk 2]
```diff
@@ -39,7 +39,7 @@
                     <div class="cert-separator"></div>
                     <div class="cert-issued-by">
                         <div>Issued under my hand at Victoria, British Columbia</div>
-                        <div>on {{ report_date_time }}</div>
+                        <div>on {{ effective_date }}</div>
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/certificateOfAmalgamation.html:42]: FYI. In incorporation filing and amalgamation filing the `effective_date` will be same as `business.formatted_founding_date`

File: legal-api/report-templates/certificateOfAmalgamation.html (hunk 1 above)
Comment (severinbeauvais) [reply legal-api/report-templates/certificateOfAmalgamation.html]: @jamespaologarcia How about a screenshot of a Certificate of Amalgamation that was done before today? Thanks.

File: legal-api/report-templates/certificateOfAmalgamation.html (hunk 2 above)
Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html]: I was not asking you change. Just want you to know both are correct. Anyways looks good to me

**Comment by severinbeauvais** [comment]
@jamespaologarcia How about a screenshot of a Certificate of Amalgamation that was done before today? Thanks.

**Comment by jamespaologarcia** [comment]
> @jamespaologarcia How about a screenshot of a Certificate of Amalgamation that was done before today? Thanks.


This is with the changes from PR:
image


This is what's currently on dev:
image

**Review by severinbeauvais — APPROVED** [review]
Excellent. The screenshots look good!

**Pull request merged**